client/me/security-ssh-key/manage-ssh-keys.tsx

```diff
@@ -46,8 +57,18 @@ const SSHKey = ( { userLogin, sshKey, onDelete, keyBeingDeleted }: SSHKeyProps )
 				</SSHKeyCard.Date>
 			</SSHKeyCard.Details>
 			<SSHKeyCard.Button
+				primary={ true }
+				scary={ false }
+				disabled={ !! keyBeingDeleted || !! keyBeingUpdated }
+				onClick={ handleUpdateClick }
+				style={ { marginInlineStart: 'auto' } }
+			>
+				{ __( 'Update SSH key' ) }
+			</SSHKeyCard.Button>
+			<SSHKeyCard.Button
+				style={ { marginInlineStart: '10px' } }
 				busy={ keyBeingDeleted === sshKey.name }
-				disabled={ !! keyBeingDeleted }
+				disabled={ !! keyBeingDeleted || !! keyBeingUpdated }
```

Comment:
Since `keyBeingUpdated` is used only to calculate if the buttons should be disabled, maybe we could pass disabled as a prop to `SSHKey` and the same variable could be used in both buttons. Line 62 and Line 71.